numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -4185,6 +4214,18 @@ initialize_numeric_types(void)
 
     /**end repeat**/
 
+
+    /**begin repeat
+     * #Type = Bool, Byte, UByte, Short, UShort, Int, UInt, Long,
+     *         ULong, LongLong, ULongLong#
+     */
+
+    Py@Type@ArrType_Type.tp_str = genint_type_str;
+    Py@Type@ArrType_Type.tp_repr = genint_type_repr;
+
+    /**end repeat**/
+
+
```

Comment:
Too many blank lines? The rest of this block is all single blanks. Not important though